.travis.yml

```diff
@@ -19,12 +19,20 @@ matrix:
   allow_failures:
     - php: 7.1
 
+before_install:
+  - composer global require hirak/prestissimo
```

Comment:
Because of the cache this is ~30 seconds. I _think_ prestissimo only does downloads, there's no map-reduce computation. of the autoloader. https://github.com/hirak/prestissimo/blob/master/src/ParallelizedComposerRepository.php